Pidgin/Parser.NoneOf.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections.Generic;
+using System.Linq;
+using Pidgin.ParseStates;
+
+namespace Pidgin
+{
+    public static partial class Parser
+    {
+        /// <summary>
+        /// Creates a parser which parses and returns a character as long as it does not match one of the specified characters.
+        /// </summary>
+        /// <param name="chars">A sequence of characters that should not be matched</param>
+        /// <returns>A parser which parses and returns a character that does not match one of the specified characters</returns>
+        public static Parser<char, char> NoneOf(params char[] chars)
```

Comment:
Up to you, I was going with the name that you suggested earlier. I do think AnyCharExcept is a more descriptive name, but it might result in more method names if we want to have various overloads that take different kinds of parsers.